src/SelfService/Application/AzureResourceApplicationService.cs

```diff
@@ -60,7 +64,45 @@ UserId requestedBy
             }
         }
 
-        var resource = AzureResource.RequestNew(capabilityId, environment, _systemTime.Now, requestedBy);
+        if (string.IsNullOrWhiteSpace(purpose))
+        {
+            throw new ArgumentException("Purpose must be provided for Azure Resource request", nameof(purpose));
+        }
+        if (purpose == "ai")
```

Comment:
eww... hardcoding values is an ugly thing.
But we could also simply not check for this ...